shunting-yard-common/src/test/java/com/hotels/shunting/yard/common/emitter/AbstractMetaStoreEventListenerTest.java

```diff
@@ -78,7 +78,7 @@
   private @Mock MetaStoreEventSerDe eventSerDe;
   private @Mock MessageTask messageTask;
   private @Mock MessageTaskFactory messageTaskFactory;
-  private @Mock SerializableListenerEventFactory serializableListenerEventFactory;
+  private @Mock SerializableListenerEventFactory SerializableListenerEventFactory;
```

Comment:
`SerializableListenerEventFactory SerializableListenerEventFactory` -> `SerializableListenerEventFactory serializableListenerEventFactory`
It's a variable's name.

Reply:
Same with the other `SerializableApiary*Event` mocked objects.